fission-server/src/db/mod.rs

```diff
@@ -0,0 +1,9 @@
+#![allow(missing_docs)]
```

Comment:
I only wanted to apply this to the schema file because it is auto generated, but couldn't figure out how to do that.

Reply:
instead of the whole module, I just tagged the schema.rs:

```rust
#[allow(missing_docs, unused_imports)]
pub mod schema;
```

Reply:
Thanks! I thought I tried that 🤔 Anyhow, works now 👍